source/stylesheets/components/_footer.scss

```diff
@@ -0,0 +1,114 @@
+//
+// Footer Component
+//
+// Based on the existing GOV.UK footer in GOV.UK Template
+
+@include exports("footer") {
+
+  .app-c-footer {
+    display: block;
+    margin-top: $govuk-spacing-scale-1;
+    padding: $govuk-gutter $govuk-gutter-half ($govuk-gutter * 2);
+    border-top: 1px solid $govuk-footer-border-top;
+    color: $govuk-footer-text;
+    background: $govuk-footer-background;
+    @include mq($from: desktop) {
+      padding: $govuk-gutter;
+    }
+
+    a:link,
+    a:visited {
+      color: $govuk-footer-link;
+    }
+
+    a:hover,
+    a:active {
+      color: $govuk-footer-link-hover;
+    }
+  }
+
+  .app-c-footer__heading {
+    margin-bottom: 0;
+    color: $govuk-text-colour;
+  }
+
+  .app-c-footer__heading:nth-of-type(1) {
+    margin-top: 0;
+  }
+
+  .app-c-footer__list {
+    margin: 0;
+
+    li {
```

Comment:
@alex-ju, how do you feel about using a descendant selector (or classes) here to make the style more specific? I've created a [card](https://trello.com/c/mEN36uTR/30-should-we-have-specific-class-names-for-all-elements-inside-components) to discuss our approach to this in general.

Reply:
Responding to the comment above this... Both the header and the footer are full width intentionally. I think it works in context for most of the Design System. The constrained layouts are probably the odd ones out and I would want the header and footer to remain consistent throughout.

Is the highlight / outline thing due to it being a block?

Reply:
@dashouse: yes, fixed the highlight, see comment below.

Reply:
@hannalaakso 👍 for adding the descendant selectors to our discussions